widgetry/src/event_ctx.rs

```diff
@@ -138,6 +138,8 @@ impl<'a> EventCtx<'a> {
     }
 
     pub fn set_style(&mut self, style: Style) {
+        *self.prerender.assets.style.borrow_mut() = style.clone();
+        self.prerender.assets.clear_text_cache();
         *self.style = style;
```

Comment:
Is it odd to keep around two copies of Style? We can be confident they're in sync because of this method. But any reason not to remove the one on `EventCtx` and use the other?

Reply:
Yup, it's weird alright!

The alternative was my initial approach - but so many things expect to access style off of ctx. Having that return a Ref instead of a &Style caused a bunch of problems with the borrow checker. I started to work through them by doing some juggling at each call site, but the verbosity and the work seemed worse than the approach I ultimately landed on.

It's certainly possible that I missed something that would have solved this in a less weird way.

Reply:
Ah, that's fine. Don't burn effort on it. Since there's no risk of these getting out of sync, a copy is fine.